pkgs/development/python-modules/pytest-xdist/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchzip, buildPythonPackage, isPy3k, execnet, pytest, setuptools_scm }:
+
+buildPythonPackage rec {
+  name = "${pname}-${version}";
+  pname = "pytest-xdist";
+  version = "1.14";
+
+  src = fetchzip {
+    url = "mirror://pypi/p/pytest-xdist/${name}.zip";
+    sha256 = "18j6jq4r47cbbgnci0bbp0kjr9w12hzw7fh4dmsbm072jmv8c0gx";
+  };
+
+  buildInputs = [ pytest setuptools_scm ];
+  propagatedBuildInputs = [ execnet ];
+
+  prePatch = ''
+    rm testing/acceptance_test.py testing/test_remote.py testing/test_slavemanage.py
+  '';
+
+  checkPhase = ''
+    py.test
```

Comment:
It's about 10 test cases in total, I didn't want to disable them all separately.